app/views/admin/videos/new.html.haml

```diff
@@ -0,0 +1,27 @@
+%section.admin_add_video
+  .container
+    .row
+      .col-md-10.col-md-offset-1
+        = bootstrap_form_for [:admin, @video] do |f|
+          %ul.nav.nav-tabs
+            %li
+              %a(href="/ui/admin_views_payments") Recent Payments
+            %li.active
+              =link_to "Add a New Video", new_admin_video_path
+          %br
+          %fieldset
+            .col-sm-6
+              = f.text_field :title, class: 'form-control'
+            .col-sm-6
+              = f.select :category_id, options_for_select(Category.all.map { |category| [category.name, category.id] })
+            .col-sm-12
+              = f.text_area :description, class: 'form_control', rows: 8
+            =f.file_field :large_cover, label: "Upload Large Cover Image"
+            =f.file_field :small_cover, label: "Upload Small Cover Image"
+            =f.text_field :video_url, class: "span3", label: "Video URL"
+              
+            %input(type="submit" value="Add Video" class="btn btn-primary add-video-btn")
+
+%script
+  $('input[type=file]').bootstrapFileInput();
+  $('.file-inputs').bootstrapFileInput();
```

Comment:
Better practice would be to have this unobtrusive way, in a separate javascript file and include it in this template via javascript_include_tag. Just like you include `payment.js` for Stripe :) This kind of functionality may be used elsewhere too.